pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -277,6 +287,19 @@ def __auto_reduce_result_objs(self, outputs):
 
         return eval_results
 
+    def on_predict_epoch_end(self):
+        self.trainer._progress_bar_callback.on_test_end(
+            self.trainer, self.trainer.get_model())
```

Comment:
```suggestion
        self.trainer._progress_bar_callback.on_test_end(self.trainer, self.trainer.get_model())
```
pls update your IDE to 120 line length not 80